duckduckgo/macos-browser
# Bump find-in-page to latest version

Task/Issue URL: https://app.asana.com/0/1200467065115223/1201394530283598/f
Tech Design URL:
CC:

**Description**:
Includes fixes for highlight positioning across a number of sites
including example.com, spreadprivacy.com and wikipedia

 - [Position within a layout causes styles to be misapplied](https://app.asana.com/0/1200467065115223/1200463636340946)
 - [Find in page highlights inherit styles from parent document](https://app.asana.com/0/1200467065115223/1200628903539962)
 - [Find in page not highlighting on spreadprivacy.com](https://app.asana.com/0/1200467065115223/1200792097969059)
 - [Search highlight on the wrong place](https://app.asana.com/0/1200467065115223/1201381511627364)

**Steps to test this PR**:
See tasks linked above for steps to recreate each issue.
Main issue fixed -- find-in-page use on SpreadPrivacy.com, e.g. try searching for something on https://spreadprivacy.com/introducing-email-protection-beta/

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
